composer.json

```diff
@@ -1,7 +1,10 @@
 {
     "name": "laravel/laravel",
     "description": "The Laravel Framework.",
-    "keywords": ["framework", "laravel"],
+    "keywords": [
```

Comment:
@theflow This needs investigation: Is this "Composer" style, or is it a decision by the package maintainer? Can we even prevent this change from happening in an automated system (unless we RegExp our way out of this)